tests/model_hub_tests/pytorch/requirements.txt

```diff
@@ -25,3 +25,32 @@ transformers
 wheel
 PyYAML
 kornia
+
+openvino-tokenizers
```

Comment:
please exclude and install tokenizers built in our GHA. See TF Hub model tests

Reply:
Fixed.